Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -448,6 +448,10 @@ extension Client {
         }
 
         // Check if coming back after triggered web flow login
+        if url.pathComponents.contains("bankId") {
+            return
```

Comment:
Unfortunately, we cannot merge this way. We are missing the completion handler here. This method needs to return LoginresultHandler which contains User after successful login or LoginError in case of failure.